dashboard/src/ui/dashboard/components/UsageLimitsPanel.jsx

```
@@ -922,6 +924,65 @@ function ArkCodingPlanSetupHint() {
+    "npm install -g @volcengine/ark-cli",
+    "arkcli auth login volc-sso   # browser SSO (recommended)",
+    "arkcli auth status           # verify the session",
+  ].join("\n");
+
+  const onCopy = async (e) => {
+    e.stopPropagation();
+    try {
+      await navigator.clipboard.writeText(snippet);
+      setCopied(true);
+      setTimeout(() => setCopied(false), 1600);
+    } catch (_e) {}
```

ESLint

[error] Empty block statement.

(no-empty)